### ISSUE TITLE
扫描后台日志报错

### ISSUE THREAD

**Issue by wzl1215** [zj1244/beholder_scanner#2]
你好：
centos7虚拟机 docker部署beholder_scnaner,并发数设置为50，扫描一个C网段与宿主机同网段的服务器，端口范围1-10000，docker logs beholder_scanner看到大量下图的报错
![捕获](https://user-images.githubusercontent.com/65029258/81418882-f5447580-917f-11ea-87be-0470f08a70cf.PNG)

**Comment by wzl1215** [comment]
[2020-05-08 22:31:09,434] [ERROR] {"ip": "10.0.10.65", "task_status": "ready", "port": "1-10000", "base_task_id": "5eb567aba8b06f000185b752"}
Traceback (most recent call last):
  File "/opt/beholder_scanner/scanner/lib/utils/common.py", line 120, in run_nmap
    nm.scan(hosts=ip, arguments='-sV -p%s -T4 --version-intensity 4' % port)
  File "/opt/beholder_scanner/scanner/lib/utils/portscan.py", line 229, in scan
    nmap_warn_keep_trace=nmap_warn_keep_trace
  File "/opt/beholder_scanner/scanner/thirdparty/nmap.py", line 327, in analyse_nmap_xml_scan
    raise PortScannerError(self._nmap_last_output)
PortScannerError: u'<?xml version="1.0" encoding="UTF-8"?>\n<!DOCTYPE nmaprun>\n<?xml-stylesheet href="file:///usr/local/bin/../share/nmap/nmap.xsl" type="text/xsl"?>\n\n<nmaprun scanner="nmap" args="nmap -oX - -sV -p1-10000 -T4 -&#45;version-intensity 4 10.0.10.65" start="1588947971" startstr="Fri May  8 22:26:11 2020" version="7.80" xmloutputversion="1.04">\n<scaninfo type="syn" protocol="tcp" numservices="10000" services="1-10000"/>\n'

**Comment by zj1244** [comment]
> [2020-05-08 22:31:09,434] [ERROR] {"ip": "10.0.10.65", "task_status": "ready", "port": "1-10000", "base_task_id": "5eb567aba8b06f000185b752"}
> Traceback (most recent call last):
> File "/opt/beholder_scanner/scanner/lib/utils/common.py", line 120, in run_nmap
> nm.scan(hosts=ip, arguments='-sV -p%s -T4 --version-intensity 4' % port)
> File "/opt/beholder_scanner/scanner/lib/utils/portscan.py", line 229, in scan
> nmap_warn_keep_trace=nmap_warn_keep_trace
> File "/opt/beholder_scanner/scanner/thirdparty/nmap.py", line 327, in analyse_nmap_xml_scan
> raise PortScannerError(self._nmap_last_output)
> PortScannerError: u'\n\n\n\n\n\n'

出现这个问题是因为nmap扫描目标主机的时间超过了5分钟，然后程序杀死了超时的nmap造成的。

**Comment by zj1244** [comment]
issue:closed by commit 4b4c705

**Comment by zj1244** [comment]
@wzl1215 你需要用如下命令重新拉取最新镜像
```
# docker-compose -f xxxxxxx.yml pull
# docker-compose -f xxxxxxx.yml up -d
```

issue:closed by commit 4b4c705
